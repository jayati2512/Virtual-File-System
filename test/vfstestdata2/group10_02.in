createvfs demo1
createvfs demo2 1024
createvfs demo3 0
mountvfs demo1
mountvfs demo2
mountvfs demo2
unmountvfs demo3
unmount
unmount demo2
