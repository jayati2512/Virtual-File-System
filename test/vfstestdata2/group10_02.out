createvfs_FAILURE VFS_INSUFFICIENT_ARGUMENTS
createvfs_SUCCESS
createvfs_FAILURE INVALID_SIZE
mountvfs_FAILURE DATA_FILE_NOT_FOUND
mountvfs_SUCCESS
mountvfs_FAILURE VFS_ALREADY_MOUNTED
unmountvfs_FAILURE VFS_NOT_MOUNTED
unmountvfs_FAILURE VFS_INSUFFICIENT_ARGUMENTS
unmountvfs_SUCCESS
